### ISSUE TITLE
Question on TinaFace training setting?

### ISSUE THREAD

**Issue by huangh12** [Media-Smart/vedadet#58]
Hi @hxcai , thanks for sharing the impressive work. I have read the TinaFace paper and been puzzled by the training setting in the paper.  The lr schedule and lr rate seems very complex and differs a lot from the official RetinaNet.  Is this training setting a careful tunning result of the experiments or being inspired by other work.    Could you reveal the motivation behind that? Thank you!

![图片](https://user-images.githubusercontent.com/18748036/123067661-c18be400-d443-11eb-8aaf-63028d8e3949.png)

**Comment by hxcai** [comment]
@huangh12 Hi，we use the restart learning scheduler to conduct experiments for convenience and do not tune carefully.

**Comment by hxcai** [comment]
@huangh12 Hi，we use the restart learning scheduler to conduct experiments for convenience and do not tune carefully.

**Comment by huangh12** [comment]
Thank you.
